examples/openai/azure_openai.py

```diff
@@ -27,7 +27,7 @@
 # Load native OpenAI model
 native_model = mlflow.openai.load_model(model_info.model_uri)
 completion = openai.ChatCompletion.create(
-    model=native_model["model"],
+    deployment_id="<YOUR AZURE OPENAI DEPLOYMENT ID (ALSO CALLED DEPLOYMENT NAME)>",
```

Comment:
Can we use an environment variable?

Reply:
This can be multiple values depending on what models you deploy, does environment variable make any convenience here?

Reply:
I'm not sure but I want to avoid pushing my deployment_id to GitHub.